docs/tutorials/sdk-tutorials/create-a-profile.md

```diff
@@ -143,4 +143,4 @@ createProfileHandler("John Doe");
 createProfileHandler("John Not Doe", true);
 ```
 
-Now that we have successfully created a profile, let's proceed to [fetch the profile](./fetch-a-profile.md)
+Voila! We have successfully created a new profile, let's proceed to [fetch the profile](./fetch-a-profile.md)
```

Comment:
I would prefer the older version: `Now that we ...` instead of `Voila`. Or maybe if you want, you can congratulate the reader for it's achievement.

Reply:
Alright, I will revert this